### PR TITLE
[v3] Layout change to remove hacky absolute styling

### PR DESCRIPTION
**Pull request by JimmyHoenderdaal**

### CODE REVIEW

**Comment by Roene-JustBetter** [comment]
What is changed? Can you add some information and maybe screenshots?

**Comment by JimmyHoenderdaal** [comment]
visually nothing changed except for mobile 
image

**Comment by JimmyHoenderdaal** [comment]
visually nothing changed except mobile which looks like this now
image

**Review by jordythevulder — APPROVED** [review]

File: resources/views/checkout/pages/credentials.blade.php [hunk 1]
```diff
@@ -6,11 +6,17 @@
 
 @section('content')
     <div class="container">
-        <x-rapidez-ct::layout class="mt-4 sm:mt-12">
+        <x-rapidez-ct::layout class="lg:items-end">
             <x-rapidez-ct::title-progress-bar :href="route('cart')" :$checkoutSteps :$currentStep :$currentStepKey>
```
Comment (jordythevulder) [inline resources/views/checkout/pages/credentials.blade.php:10]: i see a duplicate of the exact same code, maybe create a component for it?

Comment (JimmyHoenderdaal) [reply resources/views/checkout/pages/credentials.blade.php:10]: I tried to create a component for this locally but it didn't really make it cleaner,
I think we should keep it this way

**Review by jordythevulder — APPROVED** [review]

**Pull request merged**